packages/frontend-v2/pages/index.tsx

```diff
@@ -10,8 +14,9 @@ const Bomb: React.FC = () => {
 const Home: NextPage = () => {
   const [isClientSideError, setIsClientSideError] = useState(false);
 
-  const [token, setToken] = useState("");
   const [planId, setPlanId] = useState<number>();
+  const [tokenInStorage, setTokenInStorage] = useLocalStorage("token", "");
```

Comment:
it is actually very insecure to store token in localStorage (because it is accessible via JS). i know this is temp code, but just figured id point it out

Reply:
lmao yes! it's accessible by any js that runs from our domain making us vulnerable to xss, cookies fix this but gotta make sure same site attribute is set to lax/strict to avoid csrf - something to keep in mind when doing the cookie impl.